eng/jacoco-test-coverage/pom.xml

```diff
@@ -55,11 +54,27 @@
       <artifactId>azure-core-http-netty</artifactId>
       <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;current} -->
     </dependency>
+    <dependency>
```

Comment:
Do these tools have to have strong dependencies on all the libraries in the repo? Is there not a way to have the tools take the libraries as an argument when running them as opposed to being required to depend on them?

I filed issue https://github.com/Azure/azure-sdk-for-java/issues/5344 a little while ago asking questions about these tools and whether or not we had a way to remove them as a pre-step of the build and instead pull them from a package an run them instead of directly referencing everything.

Reply:
I don't know - this would be a good thing for us to investigate, as it is a nuisance.